api/services/elastigroup/gcp/schemas/elastigroupUpdate.yaml

```diff
@@ -0,0 +1,501 @@
+type: object
```

Comment:
@vlad666spot why not using the regular elastigroup schema?

Reply:
Here to explain - it's needed screenshot (in create API call some params are required - but in update API Call the same params are not required), so that's why I need to use inline but not the regular schema) 

<img width="3840" alt="37" src="https://user-images.githubusercontent.com/64230439/98139766-3b7b5700-1ecd-11eb-87d6-aa6571e1626f.png">